pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -110,13 +110,26 @@ class Build {
 
     def runTests() {
         def testStages = [:]
-        List testList = []
-
+        List testList = buildConfig.TEST_LIST
```

Comment:
Sorry to keep nitpicking here, but can some of this be extracted into small private functions?  It seems that we're doing a setJDKBranch() and a setJDKRepo()...

Reply:
Thanks @karianna! Instead of doing a set***() I prefer to do a get***().  As tests are actually getting those information from build parameters.